GPU/GPUTracking/TRDTracking/GPUTRDTrackletWord.cxx

```diff
@@ -17,6 +17,8 @@ using namespace GPUCA_NAMESPACE::gpu;
 #include <new>
 #endif
 
+#ifdef GPUCA_ALIROOT_LIB
```

Comment:
Not sure, but should this be `GPUCA_TPC_GEOMETRY_O2`, too?

Reply:
Thanks for spotting. Yes, actually the whole cxx file is only needed for AliRoot. I have cleaned that up and I will squash the commits now. FullCI should also go through. At least the compilation inside the slc8-gpu-builder was successful.